src/btree.h

```diff
@@ -176,6 +176,7 @@ typedef void (*btree_async_cb)(struct btree_async_ctxt *ctxt);
 
 // States for the btree async lookup.
 typedef enum {
+   btree_async_state_uninit = 0,
```

Comment:
I think we should be consistent about "invalid" vs. "uninit".  I think "invalid" is better because we sometimes set things to "invalid" (so sometimes we _do_ init something to the invalid state) and because uninitialized things are not guaranteed to be 0 anyway.

Reply:
@rtjohnso I corrected all instances of `_UNINIT` with `_INVALID`.